src/frontend/magnify/src/components/groups/GroupRow/mocks.ts

```diff
@@ -0,0 +1,17 @@
+const names = ['John', 'Jane', 'Bob', 'Mary', 'Jack', 'Jill', 'Adam', 'Julie', 'Bobby'];
```

Comment:
As suggested in a comment you can use fakerjs for this (the new one) https://github.com/faker-js/faker